test/Microsoft.ML.Tests/TrainerEstimators/SdcaTests.cs

```diff
@@ -16,7 +16,7 @@ public void SdcaWorkout()
             var dataPath = GetDataPath("breast-cancer.txt");
 
             var data = TextLoader.CreateReader(Env, ctx => (Label: ctx.LoadFloat(0), Features: ctx.LoadFloat(1, 10)))
-                .Read(dataPath);
+                .Read(dataPath).Cache();
```

Comment:
What would be user experience if you don't shuffle? Exception?

---
In reply to: [238761329](https://github.com/dotnet/machinelearning/pull/1780#discussion_r238761329) [](ancestors = 238761329)

Reply:
It can be 100x slower in the worst case.

---
In reply to: [238883768](https://github.com/dotnet/machinelearning/pull/1780#discussion_r238883768) [](ancestors = 238883768,238761329)